client/src/api/ImageRepository.ts

```diff
@@ -0,0 +1,56 @@
+import { ApiResponse, ID } from '../types/types.ts';
+import ImageApiPaths from './ImageApiPaths.ts';
+import { fetchData } from './fetchData.ts';
+
+interface IImageRepository {
+  getImageById: (id: ID) => Promise<ApiResponse>;
+  createImageForChat: (chatId: ID, imageData: any) => Promise<ApiResponse>;
+  createImageForUser: (userId: ID, imageData: any) => Promise<ApiResponse>;
```

Comment:
pozbylbym sie tego any, to nie jest po prostu string?

Reply:
ImageDocument